yaml/roads.yaml

```diff
@@ -94,6 +91,12 @@ global:
       incline: {col: tags->incline}
       trail_visibility: {col: tags->trail_visibility}
       sac_scale: {col: tags->sac_scale}
+  - &ne_properties
+      name: {col: name}
```

Comment:
let's add `ref: { col: name }` to the `&ne_properties` block, but not export `name: {col: name}`